BrainPortal/public/swagger/lang/it.js

```diff
@@ -26,7 +26,7 @@ window.SwaggerTranslator.learn({
     "List Operations":"Mostra operazioni",
     "Expand Operations":"Espandi operazioni",
     "Raw":"Grezzo (raw)",
-    "can't parse JSON.  Raw result":"non è possibile parsare il JSON. Risultato grezzo (raw).",
+    "can't parse JSON.  Raw result":"non è possible parsare il JSON. Risultato grezzo (raw).",
```

Comment:
Please revert all swagger code changes: the language files are probably correct and we are not responsible for them anyway.